---
layout: archive
title: "Curriculum Vitae"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

------

<span style="color:black">**Education**</span>
======
------
**Ph.D., Agronomy, Specialization in Plant Breeding and Genetics** <br/>
[University of Nebraska](https://www.unl.edu/) - Lincoln, May 2017 <br/>
* Advisor: [Dr. Stephen Baenziger](https://agronomy.unl.edu/baenziger) <br/>
* Dissertation: "Development of High-Density Linkage Map and QTL Mapping For Agronomic Traits in Bread Wheat Evaluated Across Multiple Rainfed Environments"
		
**M.S., Plant Breeding and Genetics**<br/>
[SKUAST-K](http://www.skuastkashmir.ac.in/), Srinagar, India, December 2011<br/>
* Advisor: [Dr. Gulzar Singh Sanghera](https://www.researchgate.net/profile/Gulzar_S_Sanghera)<br/>
* Dissertation: " Genetic Studies on Heterosis and Combining Ability for Hybrid Rice (<i>Oryza sativa</i>) Development Under Temperate Conditions"

**B.S., Agriculture**<br/>
[SKUAST-K](http://www.skuastkashmir.ac.in/), Srinagar, India, June 2009

 ------ 
 
<span style="color:black">**Research Experience**</span>
======
------
**Postdoctoral Research Associate**<br/>
March 2018 to Current<br/>
Department of Agronomy and Horticulture, University of Nebraska - Lincoln<br/>
* Supervisors: [Gota Morota](https://www.apsc.vt.edu/OurPeople/Faculty/GotaMorota.html) and [Harkamal Walia](https://agronomy.unl.edu/walia)<br/>
* Responsibilities:
	* Quantitative genetic analysis of longitudinal high-throughput image data of night temperature stress in wheat and rice.
	* Perform genome-wide analysis and genomic predictions of high-throughput image data combined with high dimensional genomic, transcriptomic and metabolomics data. 
	* Modelling variance-heterogeneity in genome-wide association studies using novel statistical tools.
	* Develop applications related to quantitative genetic studies.
	
**Research Associate Scientist-II (Wheat Breeder)**<br/>
May 2017 to December 2017 <br/>
Maharashtra Hybrid Seed Company ([MAHYCO](https://mahyco.com/)), Aurangabad, Jalna, India<br/>
* The goal was to develop wheat lines and hybrids using advanced statistical and genomics tools<br/>
	
**Ph.D([ICAR International Fellow](https://icar.org.in/) and Graduate Research Assistantship)**<br/>
January 2014 to April 2017 <br/>
* Focused on the development and evaluation of RIL mapping population related to drought tolerance.
* Development of high-density linkage map based on SNPs derived from genotyping-by-sequencing.
* Genome-wide QTL mapping for various agro-physiological traits phenotyped across a wide range of environments.
* High-throughput phenotyping in wheat and soybean using newly developed phenotyping platforms.
* Genome selection/predictions in Nebraska wheat breeding program.
* Gained proficiency in statistical analysis and programming including Unix operating system, perl language, GBS data analysis, and statistical analysis using R and SAS software.
* Conducted extensive field trials, crossing, phenoyping and data collections across multiple environments for the four years.
* Analyzed multi-environment phenotypic data in ASREML package in R and incorporating spatial corrections using mixed linear models and generate variance components.
* BLUP/BLUE estimates for genomic selections and QTL mapping.
* Molecular mapping of Tan spot resistance in wheat.
  	
**Research Fellow ([INSPIRE Fellowship](http://www.inspire-dst.gov.in/))**<br/>
December 2011 to December 2013 <br/>
Agricultural University, [CSKHPKV](http://www.hillagric.ac.in/), Palampur India<br/>
* Diversity analysis in oats, screening and molecular mapping of powdery mildew resistance in oats.
* Double haploid breeding in wheat using *Imperata cylindrica* grass.
	
**Graduate Research Assistant (M.S.)**<br/>
June 2009 to December 2011 <br/>
* Identification of potential restorers for newly developed temperate CMS lines and identify the heterotic combinations.
	 
 ------ 
 
<span style="color:black">**Publications**</span>
======
------
**Under Preparation**

* **Hussain, W**., Walia, H., Jarquin, D., and Morota, G. 2019. Variance-heterogeneity genome-wide mapping for cadmium in wheat (*Triticum Aestivum L.*) revealed novel genomic regions and candidate genes. Target Journal: *BMC Plant Biology*.
 
* **Hussain, W**., Belamkar, V., Guttieri, M.J., Poland, J and Baenziger, P.S. 2019. Genotyping-by-sequencing derived high-density linkage map reveled novel genomic loci and candidate genes for plant height in wheat (*Triticum Aestivum L.*) Target Journal: *The Plant Journal*. 

------

**Peer-Reviewed Articles**

------

  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
------

<span style="color:black">**Book Chapters**</span>
======
------
* Rana,M., Sood, A., **Hussain, W**., Kaldate, R., Sharma, T.R., Gill, R.K.,  Kumar, S., Singh, S. (2019). In: Mohar Singh (eds), Gene Pyramiding and Multiple Character Breeding, Lentils: Potential Resources for Enhancing Genetic Gains, 2019, Pages 83-124. [Link](https://doi.org/10.1016/B978-0-12-813522-8.00006-6).
* Sanghera, G.S., Wani, S.H.,  **Hussain, W**., and Singh, N.B. (2015). *Genetic Engineering for Cold Stress Tolerance in Crop Plants*. In book: Advances in Genome Science, Edition: Volume 4, Publisher: Bentham Science, Editors: Atta-ur-Rahman, pp. 173-201. [Link](http://www.eurekaselect.com/138849/volume/4)
* Chaudhary, H.K., Kaila, V., Rather, S.A., Badiyal, A., **Hussain, W**., Jamwal, N.S., and Mahato, A. (2013). In: Pratap and J. Kumar (eds.), *Alien Gene Transfer in Crop Plants, Volume Achievements and Impacts*. Springer, pp 1-26. [Link](https://link.springer.com/chapter/10.1007/978-1-4614-9572-7_1)
* **Hussain, W**., Sanghera, G.S., Jamawal, N.S, and Badiyal,A. (2013). *Crop improvement through genomic interventions in sustainable way*. In: Malik CP, Sanghera GS and Sharma P(ed) Crop improvement: An integrated approach. MD Publications Pvt Ltd, New Delhi. pp 61-68. ISBN 978-81-7533-456-4. [Link](https://www.cabdirect.org/cabdirect/abstract/20143204150)
* Dar, S.H., **Hussain, W**., and Sanghera, G.S. (2013). *Advances in hybrid rice technology through applications of novel technologies*. In: Malik CP, Sanghera GS and Sharma P(ed), Crop improvement: An integrated approach. MD Publications Pvt Ltd, New Delhi. pp 1-12. [Link](https://www.cabdirect.org/cabdirect/abstract/20143204150) 


<span style="color:black">**Talks**</span>
======
------
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  
------  

<span style="color:black">**Fellowships and Awards**</span>
======
------
* Open Science Grid User School Travel Grant Award 2018. [Link](https://opensciencegrid.org/user-school-2018/).
* Travel Grant Award Kansas State Plant Breeding Symposium-2017. [Link](https://www.k-state.edu/pbg/symposium2017/speakers.html)
* USDA ICQG Award-2016. [Link](https://agronomy.unl.edu/news/bandillo-mcconaughy-and-waseem-awarded-fellowships).
* Best Oral Presentation Award, UNL Plant breeding Symposium-2016. [Link](https://agronomy.unl.edu/plant-breeding-symposium)
* Indian Council of Agricultural Research (ICAR)-International Fellowship(2014-2017). [Link](https://icar.org.in/node/5142)
* Inspire Fellowship (Ministry of Science & Technology, Department of Science & Technology, India (2012-2104). [Link](http://www.inspire-dst.gov.in/)
* University Merit Certificate (M.Sc. Plant breeding &Genetics) (2011). [Link](http://www.skuastkashmir.ac.in))
* Qualified ASRB National Eligibility Test 2012, 2013 and 2014 for Assistant Professorship. [Link](http://www.asrb.org.in/)

------

<span style="color:black">**Professional Training**</span>
======
------
* Software Carpentry Workshop on computational skills, including task automation, version control, and modular programming. Learn Bash, Git and Python. Holland Computing Center, University of Nebraska, Lincoln. August 13 & 14, 2018. [Link](https://unlhcc.github.io/2018-08-13-NU/)
* Open Science Grid (OSG) User School 2018: Workshop on High Throughput Computing and Running Large Scale Computing Applications at University of Wisconsin Madison. July 9-13. [Link](https://opensciencegrid.org/user-school-2018/)
* Work shop Series on Unix Shell, Git and use of HCC’s High-Performance Computing, High Throughput Computing and Cloud computing resources. Holland Computing Center, University of Nebraska Lincoln. June 05-26, 2018. [Link](https://hcc.unl.edu/june-workshop-series-2018)
* Short term bioinformatics training (RNA-seq and DNA-seq). ArrayGen Technologies, India, Pune . 23 Oct-21 Nov. 2017. [Link](http://www.arraygen.com/workshop-training.php)
* Understanding genome-wide association studies and other big data biological applications. University of Nebraska, Lincoln. June 23-24, 2014.
* Fundamental writing skills workshop researchers. University of Nebraska, Lincoln. Sept 2015.
* Write Wining Grant Proposals, University of Nebraska, Lincoln, March 18, 2016.

------

<span style="color:black">**Open Source Contribution**</span>
======
------
* ShinyAIM: Shiny-based Application of Interactive Manhattan Plots for Longitudinal GWAS using R and Shiny. The application is available online at [https://chikudaisei.shinyapps.io/shinyaim/](https://chikudaisei.shinyapps.io/shinyaim/). The source code and materials can be downloaded from GitHub [https://github.com/whussain2/ShinyAIM](https://github.com/whussain2/ShinyAIM)

------

<span style="color:black">**Leadership Roles**</span>
======
------
* A representative of Department of Agronomy and Horticulture Safety Committee in the year 2015.
* Member of organizing a committee of UNL plant breeding symposium-2016.
* Hosted 2015- World Food Prize Nebraska Youth Institute in Department of Agronomy and Horticulture.
* Poster Judge (Undergraduate and Graduate) at University of Nebraska, Lincoln Spring Research Fair-2018.
* Poster Judge (Undergraduate) at ASA, CSSA Anual meetings in Baltimore, Maryland-2018.

------

<span style="color:black">**Professional Memberships**</span>
======
------
* ASA, CSSA,and SSSA since June-2014
* National Association of Plant Breeders since Feb. 2018
* American Society of Plant Biologists since April 2018

------

<span style="color:black">**Professional Experience**</span>
======
------
* Associate Editor: Agronomy Journal (March 2018 to present)
* Peer Reviewer: Agronomy Journal (American Society of Agronomy); PNAS, Biological Sciences, India; Physiology and Molecular Biology of Plants; Agronomy (MPDI). 

------

<span style="color:black">**Posters**</span>
======
------
* Lata, S., Guleria, S.K., Thakur, K., Kumari, R., Rana, M., and **Hussain, W**: *Introgression of Opaque2 Gene Through Marker-Assisted Backcross Breeding in Elite Maize Inbred Lines*. 13th Asian Maize Conference and Expert Consultation on Maize for Food, Feed, Nutrition and Environmental Security, Ludhiana, India October 8-10,2018. P140. [Link](https://maize.org/wp-content/uploads/sites/5/2018/09/13thAMCBookofAbstracts.pdf)
* Venegas, J.P., Graybosch, R., **Hussain, W**., Bai, G., St Amand, P., Baenziger, P.S., Blecha, S: *High-Density Linkage Map Construction and Mapping of Mutant Low Phytate QTLs in Winter Wheat (Triticum Aestivum L.) Using Genotyping-By-Sequencing (GBS)*. ASA, CSSA and SSSA Tampa, Florida, Oct. 22-25, 2017. [Link](https://scisoc.confex.com/crops/2017am/webprogram/Paper106515.html) 
* Rana, M., Verma, P., **Hussain,W**., Kaldate, R., Shikha, D., Kaachra, A., Chahota, R.K., Bhatia,S., and Sharma, T.R: *Molecular Mapping of QTLs for Drought Tolerance and Yield Traits in Lentil*. InterDrought-V, Hyderabad International Convention Center (HICC), At Hyderabad India, 21-25 Feb. 2017. [Link](http://idv.ceg.icrisat.org/wp-content/uploads/2016/06/Abstracts_Book_Printed.pdf)
* Belamkar, V.,  Guttieri, M.J., El-Basyoni, I., **Hussain, W**., Poland, J., Jarquín, D., Lorenz, A.J., Baenziger, P.B: *Translating Genomic Research into Cultivar Development in the Nebraska Wheat Program*. Plant and Animal Genome Conference XXIV, San Diego, CA; 01/2017. [Link](https://pag.confex.com/pag/xxv/webprogram/Paper26234.html)
* Sallam, A., **Hussain, W**., Belmaker, V., and Baenziger, P.S: *QTL Mapping for Traits Associated with Drought Tolerance and Combined Drought and Heat Tolerance in Seedling Winter Wheat*. Plant and Animal Genome Conference, San Diego, CA; 01/2017. [Link](https://pag.confex.com/pag/xxv/webprogram/Paper26504.html).
* **Hussain, W**.,Stephen, P.B., Belamkar, V., Guttieri, M.J., Easterly, A., Venegas, J.P., Guedira, G.B., Poland, J: *Development of High Density Linkage Map and Genome-Wide QTL Mapping for Grain Yield in Wheat Across Multiple Rainfed Environments*. ASA, CSSA and SSSA Minneapolis, Nov. 6-9, 2106. [Link](https://scisoc.confex.com/crops/2016am/webprogram/Paper99657.html)
* Sallam, A., **Hussain, W**., Belmaker, V., and Baenziger, P.S: *Molecular Genetic Dissection to Improve Seedling Drought Tolerance inn Winter Wheat Using QTL Mapping*. At: Nebraska City, USA, Conference: Plant Science Retreat-october 2016. [Link](https://www.researchgate.net/publication/309188128_Molecular_genetic_dissection_to_improve_seedling_drought_tolerance_in_winter_wheat_using_QTL_mapping)
* Kariyawasam, G., **Hussain, W**., Easterly, A., Guttieri, M.J., Belamkar, V., Venegas, J.P., Baenziger, P.B., Poland, J., Faris, J., Xu, S., Rasmussen, J., and Liu, Z: *QTL Mapping of Resistance to Tan Spot in a Winter Recombinant Inbred Line Population Derived from Cross between Harry and Wesley*. Conference: American Phytopathological Society Annual Meeting-2016, At Tampa, Florida. [Link](https://www.apsnet.org/meetings/Documents/2016_meeting_abstracts/aps2016_189.htm)
* **Hussain, W**., Baenziger, P.B., Belamkar, V., Guttieri, M.J., Easterly, A., Venegas, J.P., Guedira, G.B., Poland, J: *SNP Discovery in Wheat RIL Population Using Genotyping-by-Sequencing and Genome-Wide QTL Mapping for Plant Height*. Conference: 5th International Conference on Quantitative Genetics, At Madison, Wisconsin, USA. [Link](https://www.researchgate.net/publication/304116172_SNP_Discovery_in_Wheat_RIL_Population_Using_Genotyping-by-Sequencing_and_Genome-Wide_QTL_Mapping_for_Plant_Height)
* Belamkar, V.,  Guttieri, M.J., El-Basyoni, I., **Hussain, W**., Poland, J., Jarquín, D., Lorenz, A.J., Baenziger, P.B: *Genomic Selection Shows Promise for Improving Winter Wheat: Insights from the University of Nebraska-Lincoln Wheat Breeding Program*. Conference: 5th International Conference on Quantitative Genetics, At Madison, Wisconsin, USA. [Link](https://www.researchgate.net/publication/304116191_Genomic_selection_shows_promise_for_improving_winter_wheat_Insights_from_the_University_of_Nebraska-Lincoln_wheat_breeding_program)
* Belamkar, V.,  Guttieri, M.J., El-Basyoni, I., **Hussain, W**., Poland, J., Jarquín, D., Lorenz, A.J., Baenziger, P.B: *Integration of Genomic Selection in the Nebraska Wheat Breeding Program*. Plant and Animal Genome Conference XXIV,, San Diego, CA; 01/2016. [Link](https://www.researchgate.net/publication/290998355_Integration_of_genomic_selection_in_the_Nebraska_wheat_breeding_program)
* Hussain, W., Stephen, P.B, Guttieri, M.J., Easterly, A., Venegas, J.P., Guedira, G.B., Poland, J: *Mapping QTLs for Plant Height Variation in RIL Population Derived from Cross Between Harry X Wesley Semi-Dwarf Wheat Lines*. ASA, CSSA and SSSA Minneapolis, Nov. 15-18, 2105. [Link](https://scisoc.confex.com/crops/2015am/webprogram/Paper92536.html)

------

<span style="color:black">**Skills**</span>
======

------
* Quantitative Skills
  * Genotyping-by-sequencing data analysis and SNP calling
  * Quantitative Genetics
    * QTL Mapping: R/qtl, R/ASMap, IciMapping etc.
    * Genome-wide Association Mapping: Plink, TASSEL, BGLR, rrBLUP, GAPIT, FarmCPU etc. 
    * Genomic Selections: BGLR,rrBLUP etc.
  * Statistics
    * SAS
    * R
    * ASReml
    * Python

* Wet Lab Skills
	*  DNA extractions, PCR, real-time PCR, electrophoresis, SSR marker and KASP marker genotyping
* Computational Skills
	* Operating Systems: LINUX, OS X, Windows
	* Big data analysis, high-throughput and cloud computing, linux shell and git
	* LATEX and R Markdown
	* Perl

------

<span style="color:black">**Teaching Experience**</span>
======
------
[**Teaching Experience 1**](https://whussain2.github.io/teaching/2018-ASCI%20944/STAT%20844%20Quantitative%20Methods%20for%20Genomics%20of%20Complex%20Traits)

* This lecture was one of the lecture in course **ASCI 944/STAT 844 Quantitative Methods for Genomics of Complex Traits**  taught by [Dr. Gota Morota](http://morotalab.org/). In this lecture I talk about **basic concepts in GWAS**, **how to correct for population stratification**, and **statistical models to run GWAS** and also showed **practical demonstration** how to perform **GWAS in R-software**.

[**Teaching Experience 2**](https://whussain2.github.io/teaching/2018-Rclub-Lecture)

* This lecture was delivered in Rclub hosted by students in the Agronomy and Horticulture department, University of Nebraska, Lincoln. In this lecture I discussed about **data visualizations in ggplot**, **interactive visualizations using plotly**, **multiple plots in same panel**, **correlations** and **correlation heat maps**. 


------

