---
title: "[Publications](https://scholar.google.com/citations?user=ECbwMvkAAAAJ&hl=en)"
permalink: /publications/
author_profile: true
---

<br>

## Peer-reviewed Publications
![Kariyawasam2018](http://whussain2.github.io/images/kariyawasam_2018.jpg){: .align-right height="200px" width="191px"}
<b>[Identification of quantitative trait loci conferring resistance to tan spot in a bi-parental population derived from two Nebraskan hard red winter wheat cultivars.](http://whussain2.github.io/publications/2018-11-01-Identification-of-quantitative-trait-loci-conferring-resistance-to-tan-spot-in-a-bi-parental-population-derived-from-two-Nebraskan-hard-red-winter-wheat-cultivars)</b> Kariyawasam, G.K., <b>W. Hussain</b>, A. Easterly, M. Guttieri, V. Belamkar, J. Poland, J. Venegas, S. Baenziger, F. Marias, J.B. Rasmussen, and Z.(2018). <i>Molecular Breeding</i>.

This paper characterizes the Tan spot resistance in Harry and Wesley cultivars and reveals high resistance to tan spot in Wesley cultivar is largely due to lack of two *Ptr ToxA-Tsn1* and *Ptr ToxC-Tsc1* tan spot sensitivity genes.

<br>

![Hussain2018](http://whussain2.github.io/images/hussain_2018_plantdirect.jpg){: .align-right height="200px" width="191px"}

<b>[ShinyAIM: Shiny-based application of interactive Manhattan plots for longitudinal genome-wide association studies.](http://whussain2.github.io/publications/2018-10-08-ShinyAIM-Shiny-based-application-of-interactive-Manhattan-plots-for-longitudinal-genome-wide-association-studies)</b> <b>W. Hussain</b>, M. Campbell, H. Walia, and G. Morota. (2018). <i>Plant Direct</i>.

ShinyAIM is a nice tool to visualize the Manhattan plots in interactive and easy way. The application has unique features to interpret and view dynamically GWAS outputs particularly from temporal GWAS studies.

<br>

![Sallam2018](http://whussain2.github.io/images/sallam_2018_euphytica.jpg){: .align-right height="200px" width="191px"}

<b>[Genetic variation in drought tolerance at seedling stage and grain yield in low rainfall environments in wheat.](http://whussain2.github.io/publications/2018-08-01-Genetic-variation-in-drought-tolerance-at-seedling-stage-and-grain-yield-in-low-rainfall-environments-in-wheat)</b> A. Sallam, A. Mourad, <b>W. Hussain</b>, P. Baenziger.(2018).<i> Euphytica</i>.

The manuscript describes the newly developed protocol to identify the drought genotypes in winter wheat at seedling stage in the green house and compare it across the field trials.  

<br>

![Hussain2018](http://whussain2.github.io/images/hussain_2018_reg.jpg){: .align-right height="200px" width="186px"}

<b>[Registration of a bread Wheat recombinant inbred line mapping population derived from a cross between ‘Harry’ and ‘Wesley’.](http://whussain2.github.io/publications/2018-07-02-Registration-of-a-bread-wheat-recombinant-inbred-line-mapping-population-derived-from-a-cross-between-Harry-and-Wesley)</b> <b>W. Hussain </b>, M.J. Guttieri, V. Belamkar, J. Polland, A.Sallam, and P. Baenziger.(2018). <i>Journal of Plant Registrations</i>.

 The manuscript describes the development and registration of Harry x Wesley derived recombinant inbred line mapping population. The mapping population is an important germplasm resource to breed for drought tolerance and wider adaptation in the US Great Plains.  

<br>

![Belamkar2018](http://whussain2.github.io/images/belamkar_2018_g3.jpg){: .align-right height="200px" width="186px"}

<b>[Genomic selection in preliminary yield trials in a winter wheat breeding program.](http://whussain2.github.io/publications/2018-06-19-Genomic-selection-in-preliminary-yield-trials-in-a-winter-wheat-breeding-program)</b> Belamkar, V., M.J. Guttieri, <b>W. Hussain</b>, D. Jarquin, I. Ei-basyoni, J. Polland, A.J. Lorenz, and P. Baenziger.(2017). <i>G3</i>.

The manuscript provides insights and assesses the the integration of genomic selection for grain yield in Nebraska wheat breeding programe.  


<br>

![Jamwal2018](http://whussain2.github.io/images/jamwal_2018_currentmicro.jpg){: .align-right height="200px" width="186px"}

<b>[Success in inducing effective rooting in triticale x wheat and wheat x wheat derived haploid plantlets in cocopeat mixture.](http://whussain2.github.io/publications/2018-05-10-Success-in-inducing-effective-rooting-in-triticale-x-wheat-and-wheat-x-wheat-derived-haploid-plantlets-in-cocopeat-mixture)</b> N.S. Jamwal, H.K. Chaudhary, A. Badiyal, <b> W. Hussain</b> , P. Katoch, N. Kumar.(2018). <i>International Journal of Current Microbiology and Applied Sciences</i>.

The manuscript reveals the success of  the Cocopeat mixture in inducing rooting in Triticale derived wheat haploid plants. 

<br>

![Hussain2017](http://whussain2.github.io/images/hussain_2017_sr.jpg){: .align-right height="200px" width="186px"}

<b>[Genotyping-by-sequencing derived high-density linkage map and its application to QTL mapping of flag leaf traits in bread wheat](http://whussain2.github.io/publications/2017-11-27-Genotyping-by-sequencing-derived-high-density-linkage-map-and-its-application-to-QTL-mapping-of-flag-leaf-traits-in-bread-wheat)</b> <b>W. Hussain </b>,  P. Baenziger, V. Belamkar, M.J. Guttieri, J.P. Venegas, A. Easterly, J. Polland, A.Sallam. (2018). <i>Scientific Reports</i>.

 In this manuscript we describes the development of high-density linkage map in Harry x Wesley derived RIL population, confirm the accuracy, reliability and validity of the map, and also its application in mapping QTLs for flag leaf traits.

<br>

![Bai2016](http://whussain2.github.io/images/bai_2016_computers_electronics_agriculture.jpg){: .align-right height="200px" width="186px"}

<b>[A multi-sensor system for high throughput field phenotyping in soybean and wheat breeding.](http://whussain2.github.io/publications/2016-10-14-A-multi-sensor-system-for-high-throughput-field-phenotyping-in-soybean-and-wheat-breeding) </b>B. Geng, Y. Ge, <b>W. Hussain</b>, P.S. Baenziger, G.Graef.(2017). <i>Computers and Electronics in Agriculture</i>.

The manuscript reports the development of a new multi-sensor system for high throughput phenotyping in plant breeding. 

<br>
![Jamwal2016](http://whussain2.github.io/images/jamwal_2016_acta.jpg){: .align-right height="200px" width="186px"}
<b>[Factors influencing crossability among triticale and wheat and its subsequent effect along with hybrid necrosis on haploid induction.](http://whussain2.github.io/publications/2016-04-02-Factors-influencing-crossability-among-triticale-and-wheat-and-its-subsequent-effect-along-with-hybrid-necrosis-on-haploid-induction)</b> Jamwal, N.S. , H.K. Chaudhary, A. Badiyal, <b> W. Hussain</b>.(2016). <i>Acta Agriculturae Scandinavica, Section B—Soil & Plant Science</i>.

The manuscript investigates the influence of genotypic and environmental factors on the crossability of elite triticale and wheat genotypes, screen the effect of necrotic genes in the F1 hybrids and the relative performance of hybrids towards haploid induction.  

<br>
![Sood2016](http://whussain2.github.io/images/sood_2016_pnas_india.jpg){: .align-right height="200px" width="186px"}
<b>[Genetic diversity of genus *Avena* from North Western-Himalayas Using molecular markers.](http://whussain2.github.io/publications/2016-03-01-Genetic-diversity-of-genus-Avena-from-North-Western-Himalayas-using-molecular-markers)</b>Sood,V.K., I. Rana, <b> W.Hussain</b>, H.K. Chaudhary.(2016). <i>Proceedings of the National Academy of Sciences, India - Section B: Biological Sciences </i>.

Here, we assess the diversity of wild relatives and cultivars of geneus Avena using RAPD and SSR markers.  

<br>

![Badiyal2016](http://whussain2.github.io/images/badiyal_2016_agriculture.jpg){: .align-right height="200px" width="186px"}
<b>[Comparative assessment of different auxin analogues on haploid induction in triticale x wheat derived backcross generations.](http://whussain2.github.io/publications/2016-06-01-Comparative-assessment-of-different-auxin-analogues-on-haploid-induction-in-triticale-x-wheat-derived-backcross-generations)</b> Badiyal, A., H.K. Chaudhary, N.S. Jamwal, A.K. Bhat, <b> W. Hussain</b>.(2017). <i>Agriculture Research Journal</i>.

The manuscript studies the effect of different possible auxin analogues like 2,4-D, NAA, IAA and IBA for their influence on haploid induction in triticale x wheat backcross generations.
 
<br>

![Chaudhary2015](http://whussain2.github.io/images/chaudhary_2015_range_management.jpg){: .align-right height="200px" width="186px"}
<b>[Genetic analysis for grain and forage yield and its component traits in genus *Avena* under North western Himalayas.](http://whussain2.github.io/publications/2015-05-02-Genetic-analysis-for-grain-and-forage-yield-and-its-component-traits-in-genus-Avena-under-North-western-Himalayas)</b> L. Chaudhary, V.K.Sood, <b>W. Hussain</b>.(2015). <i>Range Management and Agroforestry</i>.

Here, we describe the nature and magnitude of gene effects for forage and grain yield traits in oat.

<br>
![Badiyal2014](http://whussain2.github.io/images/badiyal_2014_cereal_research_communication.jpg){: .align-right height="200px" width="186px"}
<b>[Interactive genotypic influence of triticale and wheat on their crossability and haploid induction under varied agroclimatic regimes.](http://whussain2.github.io/publications/2014-06-15-Interactive-genotypic-influence-of-triticale-and-wheat-on-their-crossability-and-haploid-induction-under-varied-agroclimatic-regimes)</b> Badiyal, A. , H.K. Chaudhary, N.S. Jamwal, <b>W. Hussain</b>, A. Mahato, A.K. Bhat. (2014). <i>Cereal Research Communications</i>.

Here, we studied the response of diverse genotypes of triticale and wheat and environment on their crossability as well as to evaluate the efficiency of *Imperata cylindrica*-mediated chromosome elimination approach for haploid induction in triticale × wheat (*Triticum aestivum*) hybrids.

<br>
![Sanghera2012](http://whussain2.github.io/images/sanghera_2012_notulae.jpg){: .align-right height="200px" width="186px"}
<b>[Heterosis and combining ability estimates using line x tester analysis to develop rice hybrids for temperate conditions.](http://whussain2.github.io/publications/2012-08-30-Heterosis-and-combining-ability-estimates-using-line-x-tester-analysis-to-develop-rice-hybrids-for-temperate-conditions)</b> Sanghera, G.S.,<b> W. Hussain</b>. (2012). <i>Notulae Scientia Biologicae</i>.

Here, we studied the heterosis and combining ability by crossing 18 lines (Males) with two cytoplasmic newly developed male sterile (CMS) lines (Females).

<br>
![Hussain2012](http://whussain2.github.io/images/hussain_2012_electronic_journal.jpg){: .align-right height="200px" width="186px"}
<b>[Exploitation of heterosis in rice (*Oryza sativa L.*) using CMS system under temperate conditions.](http://whussain2.github.io/publications/2012-03-05-Exploitation-of-heterosis-in-rice-using-CMS-system-under-temperate-conditions)<b>Hussain, W.</b>,Sanghera, G.S.(2012). <i>Electronic Journal of Plant Breeding</i>.

Here, we reported the identification of potential restorers for the newly developed CMS lines SKAU7A and SKAU11A under temperate conditions.

<br>
![Sanghera2011](http://whussain2.github.io/images/sanghera_2011_current_genomics_journal.jpg){: .align-right height="200px" width="186px"}
<b>[Engineering cold stress tolerance in crop plants.](http://whussain2.github.io/publications/2011-01-01-Engineering-cold-stress-tolerance-in-crop-plants)</b> Sanghera, G.S., Wani, S.H.,<b>Hussain, W.</b>,Singh, N.B.(2011). <i>Current Genomics</i>.
In this paper we review the various mechanisms and genes involved in cold acclimatization and the possibilities where transgenic technology has been explored for breeding cold tolerance in crop plants.

<br>
![Sanghera2011](){: .align-right height="200px" width="186px"}
<b>[The magic of heterosis: new tools and complexities.](http://whussain2.github.io/publications/2011-02-02-The-magic-of-heterosi-new-tools-and-complexities)</b> Sanghera, G.S., Wani, S.H.,<b>Hussain, W.</b>,W. Shafi, A. Haribhushan, Singh, N.B.(2011). <i>Nature and Science</i>.
In this paper we review the moder concept of heterosis, and genetic and molecular basis of heterosis.
